### PR TITLE
Users can recommend a television show

### PR DESCRIPTION
**Pull request by adarsh**
Reason for change:
- Allow users to recommend television shows

What the change was:
- When logged out, guests should see the landing page (`pages/landing`)
- When logged in, users see the form to add a favorite TV show.
- Add `TelevisionShow` model, resources etc. to store them.
- Tests.

### CODE REVIEW

File: app/controllers/homes_controller.rb [hunk 1]
```diff
@@ -1,4 +1,9 @@
 class HomesController < ApplicationController
   def show
+    if !signed_in?
```
Comment (gabebw) [inline app/controllers/homes_controller.rb]: What if we extracted `signed_out?` to `ApplicationController`?

File: spec/factories.rb [hunk 2]
```diff
@@ -1,4 +1,8 @@
-FactoryGirl.define do
+FactoryGirl.define do
```
Comment (gabebw) [inline spec/factories.rb]: Is there an extra space at the end of this line?

File: db/migrate/20160116222402_create_television_shows.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class CreateTelevisionShows < ActiveRecord::Migration
+  def change
+    create_table :television_shows do |t|
+      t.string :name, null: false
```
Comment (gabebw) [inline db/migrate/20160116222402_create_television_shows.rb]: This should be title.

Comment (gabebw) [inline db/migrate/20160116222402_create_television_shows.rb]: Support for `title` over `name`: http://www.imdb.com/chart/toptv/

File: app/controllers/homes_controller.rb (hunk 1 above)
Comment (adarsh) [inline app/controllers/homes_controller.rb]: yes 881720a

File: app/controllers/application_controller.rb [hunk 4]
```diff
@@ -2,4 +2,8 @@ class ApplicationController < ActionController::Base
   include Monban::ControllerHelpers
 
   protect_from_forgery with: :exception
+
+  def signed_out?
```
Comment (gabebw) [inline app/controllers/application_controller.rb:8]: Let's make this a `private` method so it's not exposed in the controller.

File: db/migrate/20160116222402_create_television_shows.rb (hunk 3 above)
Comment (adarsh) [inline db/migrate/20160116222402_create_television_shows.rb]: done in1dce3c6

File: app/controllers/application_controller.rb (hunk 4 above)
Comment (adarsh) [inline app/controllers/application_controller.rb:8]: Done 3d0dbe8

**Comment by gabebw** [comment]
When CI passes :8ball:

**Pull request merged**